.eslintignore

```diff
@@ -1,7 +1,8 @@
-; ignore the submodules
+# ignore the submodules
 gutenberg
 block-experiments
 bundle
 jetpack
 bin
-
+# ignore the iOS XCFramework scaffolding
+**/ios-xcframework
```

Comment:
While working on https://github.com/wordpress-mobile/gutenberg-mobile/pull/5709, I noticed that the linters picked up errors for some of the files in here. Since non of the contents are used in the actual codebase, I decided to ignore them.

#4995